test-clients/src/main/java/com/hedera/services/bdd/suites/perf/token/TokenTransfersLoadProvider.java

```diff
@@ -179,7 +177,6 @@ private HapiApiSpec runTokenTransfers() {
 						spec.tryReinitializingFees();
 					}));
 				}
-				initializers.add(reduceFeeFor(CryptoTransfer, 2L, 3L, 3L));
```

Comment:
Why is this line removed? Will this not cause any issues in perf tests ?

Reply:
The `reduceFeeFor(...)` operation may cause the random fee initialization corruption issue if running in middle of tests with multiple clients. That's why I add the `AdjustFeeScheduleSuite` to do the fee reduction at the beginning to avoid it.